pkg/k8s/informer/informer.go

```diff
@@ -34,7 +35,7 @@ func init() {
 		utilRuntime.PanicHandlers,
 		func(r interface{}) {
 			// from k8s library
-			if r == http.ErrAbortHandler {
+			if errors.Is(r.(error), http.ErrAbortHandler) {
```

Comment:
It seems that [r is the return value of recover()](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/runtime/runtime.go#L46-L49) and has no guarantee to be an `error`.

```suggestion
			if err, ok := r.(error); ok && errors.Is(err, http.ErrAbortHandler) {
```

Reply:
good catch, thanks :100:

Reply:
just a question, how can I commit suggestion with sign-off together to by pass the check ?

Reply:
@sayboras I've never been able to get that to work, so I'm not sure either. :shrug: